### DISCUSSION TITLE
Switching to Classic score display while playing/watching causes DefaultScoreCounter to overlap other UI elements

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#15806] [Q&A] [answered]
I initially mentioned in #14770.
As the title suggests, the DefaultScoreCounter overlaps the DefaultAccuracyCounter and DefaultComboCounter when switching from Standardised to Classic score display when playing/watching using the "osu! (triangles)" skin. While I cannot show this in the skin editor (when entering, it creates a new modified skin, therefore reloading the skin and solving the problem), by creating a modified skin in Standardised score display, this behavior can be replicated, as modified skins do not change based on the score display mode.
Version 2021.1120.0-lazer
![osu_2021-11-25_14-04-20](https://user-images.githubusercontent.com/74262407/143504081-b3e8c7c6-4814-4939-b7f6-c3a563ded6a1.png)
![osu_2021-11-25_14-05-09](https://user-images.githubusercontent.com/74262407/143504087-1a730209-8af0-4d48-b3e8-6a66c7f64eb8.png)

**Comment by peppy** [comment, marked as answer]
Tracking via https://github.com/ppy/osu/issues/16067.
